release-notes/General/General_Feature_Release_10.5/General_Feature_Release_10.5.1.md

```diff
@@ -160,7 +160,12 @@ To swarm an element, users will also need config rights on the element.
 
 When Swarming has been enabled, you can swarm elements **in DataMiner Cube** via *System Center* > *Agents* > *Status*. On that page, the *Swarming* button will be displayed instead of the *Migration* button. Clicking the button will open a window where you can select the elements you want to swarm and the destination DMA.
 
-Swarming elements is also possible **via Automation**. For this, you will to write a script with two input parameters: one for the element key, i.e. the DMA ID/element ID pair of the element you want to swarm, and another for the target Agent ID. For example:
+Swarming elements is also possible **via Automation**. For this, you will to write a script with two input parameters:
```

Comment:
_"you will to write"_ => _"you will have to write"_ ?

Also, it might not be fully clear that the input parameters are relevant only for the specific example that is provided. For example, a script with no or other input parameters can still call the swarming API as long as it can somehow get hold of element ids for elements it wants to swarm.
To go even further, code other than automation scripts can also use these APIs (QAction or external tools)

Reply:
Should be clearer now.